scipy/stats/_continuous_distns.py

```diff
@@ -6123,6 +6144,25 @@ def _stats(self, b, moments='mv'):
     def _entropy(self, c):
         return 1 + 1.0/c - np.log(c)
 
+    def fit(self, data, *args, **kwds):
+        parameters = _check_fit_input_parameters(self, data, args, kwds)
+        data, fshape, floc, fscale = parameters
+        if floc is None:
+            raise FitDataError(msg="`floc` must be fixed to obtain a "
```

Comment:
@swallan: On future review, we're reversing the decision on this for now. If `floc` is not fixed, call the `super` implementation instead of raising an error. You don't need the changes to `FitDataError` above anymore.

Reply:
To clarify - you mean not to change `x` to `(x - loc) / scale`?

Reply:
Oops, no - I meant you don't need to add `msg` to it because you're not going to use that feature anymore. Please do change `x` to `(x-loc)/scale`, though.